src/libraries/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -194,9 +194,9 @@ public void DeleteAfterEnumerate_TimesStillSet()
 
             DateTime afterTime = DateTime.UtcNow.AddSeconds(1);
 
-            // Deleting doesn't change any info state
             info.Delete();
-            ValidateSetTimes(info, beforeTime, afterTime);
+            Assert.NotInRange(info.LastAccessTimeUtc, beforeTime, afterTime);
+            Assert.Equal(info.LastAccessTimeUtc, DateTime.FromFileTimeUtc(0));
```

Comment:
The two arguments here should be swapped; the first is expected, the second is actual.